Shaders/HighPassSharpen.fx

```diff
@@ -14,163 +13,112 @@ uniform float HighPassSharpOffset <
 	ui_type = "drag";
 	ui_min = 0.00; ui_max = 1.00;
 	ui_tooltip = "Additional adjustment for the blur radius. Values less than 1.00 will reduce the radius limiting the sharpening to finer details.";
-	ui_step = 0.20;
 > = 1.00;
 
 uniform int HighPassBlendMode <
```

Comment:
I think a combo box is better suited here. So `ui_type = "combo"` and `ui_items = "Soft Light\0Overlay\0Mulitply\0Hard Light\0Vivid Light\0Screen\0Linear Light"`. Only that the value would go from 0 to 6 then and not 1 to 7.